source/game/field/obj/ObjectWoodbox.hh

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include "game/field/obj/ObjectBreakable.hh"
+
+namespace Field {
+
+class ObjectWoodbox : public ObjectBreakable {
+public:
+    ObjectWoodbox(const System::MapdataGeoObj &params);
+    ~ObjectWoodbox() override;
+
+    void init() override;
+
+    /// @addr{0x8077ED7C}
+    [[nodiscard]] const char *getKclName() {
+        return "ironbox"; // woodbox when not in TTs
+    }
+
+    void calcCollisionTransform() override;
+    void enableCollision() override;
+
+private:
+    void calcFloor();
+
+    static constexpr f32 HALF_SIZE = 100.0f;
+    static constexpr f32 GRAVITY = 2.0f;
```

Comment:
We put `static constexpr`s after all other variable declarations, so move below `m_downwardsVelocity`